bin/lms-rtparser.php

```diff
@@ -228,6 +228,7 @@ function ($value) {
 );
 $rtparser_use_seen_flag = ConfigHelper::checkValue(ConfigHelper::getConfig('rt.imap_use_seen_flag', true));
 $rtparser_folder = ConfigHelper::getConfig('rt.imap_folder', 'INBOX');
+$rtparser_move_to_trashbin = ConfigHelper::getConfig('rt.move_to_trashbin', true);
```

Comment:
Popracuj nad bardziej eleganckim brzmieniem nowego ustawienia, bo te obecne brzmi za bardzo podwórkowo :)

Reply:
`imap_move_mail_to_trash` ?

Reply:
A co to ma konkretnie robić?

Reply:
przenosić maila do kosza po zaczytaniu do helpdeska

Reply:
To być może trzeba by użyć do tego ustawienia rt.imap_use_seen_flag i możliwe, że zmienić jego nazwę tak, żeby wskazywała, że chodzi o akcję na poście jaka ma być wykonana po przetworzeniu posta.
Kiedyś komuś prywatnie robiłem skrypt, który miał tak:

$mail_handled_post_action = ConfigHelper::getConfig('documents.mail_handled_post_action', 'seen');

i obsługiwałem akcje `seen` oraz `delete`. Może akcja `trash` by się przydała.

Reply:
Pomysł nawet ok - ale trzeba na końcu dołożyć jeszcze expunge() na końcu połączenia.

Reply:
@chilek może być tak?

Reply:
a może to też wrzucisz to uaktualnie wiki?

Reply:
To jest nadal w postaci gaszenia pożaru

Reply:
Zrobiłem wg zaleceń. Co mogę tam usprawnić?

Reply:
Udało mi się jeszcze zoptymalizować ustawianie flag dla wiadomości - bo da się ustawić dwie flagi na raz.
Co do expunge - da się to zrobić po cichu przy zamykaniu połączenia więc też nie trzeba robić tego oddzielnie.
Więcej usprawnień nie widzę mimo, że patrzę i patrzę.